Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -1215,6 +1219,60 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         }
     }
 
+    lazy var failureRecoveryHandler: FailureRecoveryHandling = FailureRecoveryHandler(deviceManager: deviceManager)
+
+    private func startServerFailureMonitor() async {
+        if await serverFailureMonitor.isStarted {
+            await serverFailureMonitor.stop()
+        }
+
+        if isSubscriptionEnabled, await isEntitlementInvalid() {
+            return
+        }
```

Comment:
I just noticed we're doing this for all monitors, but I'd 100% remove this as this is not where we should be checking there's an active subscription.

The tunnel shouldn't even start without a subscription.

Once the tunnel starts (for whatever reason) the monitor should start too.